File: Easy/Python/125. Valid Palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        s = s.lower()
        s.replace(" ", "")
        # Joins the string's alphanumeric characters together
        s = "".join([char for char in s if char.isalnum()])
        start = 0
        end = len(s) - 1
        
        while start <= end:
            if s[start] != s[end]:
                return False
            start += 1
            end -= 1
        
        return True

File: Easy/JavaScript/125. Valid Palindrome.js
var isPalindrome = function(s) {
    s = s.toLowerCase().replaceAll(/[^0-9a-zA-Z]+/gmi,"");
    let start = 0;
    let end = s.length - 1;
    while(start <= end){
        if(s.charAt(start) != s.charAt(end))
            return false;
        start++;
        end--;
    }
    return true;
};
